packages/client/src/arpa_reporter/components/PageNavigation.vue

```diff
@@ -220,9 +220,9 @@ export default {
     },
     navLinkClass(to) {
       if (this.$route.path === to) {
-        return 'nav-link usdr-tab-nav-link active';
+        return 'nav-link usdr-link active';
       }
-      return 'nav-link usdr-tab-nav-link';
+      return 'nav-link usdr-link';
```

Comment:
Note to reviewer:  renamed the class to be general as it only contains color setting.